aws_lambda_builders/utils.py

```diff
@@ -93,10 +92,11 @@ def _access_check(fn, mode):
 
         return None
 
-    if path is None:
-        path = os.environ.get("PATH", os.defpath)
-    if not path:
-        return None
+    path = os.environ.get("PATH", os.defpath)
+
+    if additional_search_paths:
+        for pth in additional_search_paths:
+            path.insert(0, pth)
```

Comment:
Also, it is confusing here because looks like `path` is a string and additional_search_paths is a list. I would first convert path to list before concatenating